packages/rrweb/src/record/constructableStyleSheets.d.ts

```diff
@@ -0,0 +1,10 @@
+// This informs the TS compiler about constructed stylesheets.
+// It can be removed when this is fixed: https://github.com/Microsoft/TypeScript/issues/30022
+declare interface DocumentOrShadowRoot {
+  adoptedStyleSheets: CSSStyleSheet[];
```

Comment:
Since this is not always available (no current or pre-release version of safari supports it for example) it might make sense to add it as an 'optional' parameter
```suggestion
  adoptedStyleSheets?: CSSStyleSheet[];
```